Components for metromap added (#3996)
nf-core/website: sites/docs/src/content/docs/community/brand/workflow-schematics.md

Reviewer: ```suggestion
To open the nf-core component library:

1. Go to [this URL](https://raw.githubusercontent.com/nf-core/website/refs/heads/main/sites/docs/src/assets/images/graphic_design_assets/workflow_schematics_components/generic/nf-core_components.xml) and press Ctrl+S / Cmd+S to save the file
1. Go to [draw.io](https://draw.io/)
1. Click *File → Open Library from → Device*
1. Select the downloaded the file

This will load the nf-core shapes into the sidebar without any CORS issues.
```

Reviewer: ```suggestion
```

| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | ![Metromap style pipeline workflow components](../../../../assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg) | Components for a metro-map style pipeline workflow | [SVG](https://raw.githubusercontent.com/nf-core/website/main/sites/docs/src/assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg) <br> [PDF](https://raw.githubusercontent.com/nf-core/website/main/sites/docs/src/assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.pdf) <br> [DRAW.IO](https://app.diagrams.net/?src=about#Uhttps%3A%2F%2Fraw.githubusercontent.com%2Fnf-core%2Fwebsite%2Frefs%2Fheads%2Fmain%2Fpublic%2Fimages%2Fgraphic_design_assets%2Fworkflow_schematics_components%2Fgeneric%2Fmetromap_style_pipeline_workflow_components.drawio) | James A. Fellows Yates, Maxime Garcia, Louis Le Nézet & nf-core; under a [CC0](https://creativecommons.org/publicdomain/zero/1.0/?ref=chooser-v1) license (public domain) |

Reviewer: ```suggestion
| ![Metromap style pipeline workflow components](../../../../assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg) | Components for a metro-map style pipeline workflow | [SVG](https://raw.githubusercontent.com/nf-core/website/main/sites/docs/src/assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.svg) <br> [PDF](https://raw.githubusercontent.com/nf-core/website/main/sites/docs/src/assets/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.pdf) <br> [DRAW.IO](https://raw.githubusercontent.com/nf-core/website/refs/heads/main/public/images/graphic_design_assets/workflow_schematics_components/generic/metromap_style_pipeline_workflow_components.drawio) | James A. Fellows Yates, Maxime Garcia, Louis Le Nézet & nf-core; under a [CC0](https://creativecommons.org/publicdomain/zero/1.0/?ref=chooser-v1) license (public domain) |
```
